internal/cmd/operator-sdk/generate/packagemanifests/cmd.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 The Operator-SDK Authors
+// Copyright 2021 The Operator-SDK Authors
```

Comment:
Only newly added files should have 2021 license headers.
```suggestion
// Copyright 2020 The Operator-SDK Authors
```